doc/modules/model_persistence.rst

```diff
@@ -85,8 +86,42 @@ same range as before.
 
 Since a model internal representation may be different on two different
 architectures, dumping a model on one architecture and loading it on
-another architecture is not supported.
+another architecture is not a supported behaviour, even if it might work
+on some cases.
+To overcome the issue of portability, pickle models are often deployed in
+production using containers, like docker.
 
 If you want to know more about these issues and explore other possible
 serialization methods, please refer to this
-`talk by Alex Gaynor <https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`_.
+`talk by Alex Gaynor
+<https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`_.
+
+Interoperable formats
+---------------------
+
+For reproducibility and quality control needs, when different architectures
+and environments should be taken into account, exporting the model in
+`Open Neural Network
+Exchange <https://onnx.ai/>`_ format or `Predictive Model Markup Language
+(PMML) <http://dmg.org/pmml/v4-4-1/GeneralStructure.html>`_ format
+might be a better approach than using `pickle` alone.
+
+ONNX is a binary serialization of the model. It has been developed to improve
+the usability of the interoperable representation of data models.
+It aims to facilitate the conversion of the data
+models between different machine learning frameworks, and to improve their
+portability on different computing architectures. More details are available
+from the `ONNX tutorial <https://onnx.ai/get-started.html>`_.
+To convert scikit-learn model to ONNX a specific tool `sklearn-onnx
+<http://onnx.ai/sklearn-onnx/>`_ has been developed. 
+
+PMML is an extension of the `XML
+<https://fr.wikipedia.org/wiki/Extensible_Markup_Language>`_ document standard
```

Comment:
This link is in french wikipedia

Reply:
Sorry, I've checked both in English and in French and linked the bad one: fixed now.